src/service/url-replacements-impl.js

```diff
@@ -245,6 +245,12 @@ export class GlobalVariableSource extends VariableSource {
         if (!clientIds) {
           clientIds = Object.create(null);
         }
+
+        // A temporary work around to extract Client ID from _ga cookie. #5761
```

Comment:
I think the real TODO here is to replace the config with a filter instead of this hack.